snowshu/core/main.py

```diff
@@ -82,16 +82,23 @@ def source(filename):
               default=None,
               help="Overrides the replica name found in replica.yml")
 @click.option(
-    '--barf',
+    '--barf', '-b',
     is_flag=True,
     help="outputs the source query sql to a local folder snowshu_barf_output")
+@click.option(
```

Comment:
Probably should save the parameters implementation for the subsequent ticket as discussed